### PR TITLE
Update of org and repo secrets: clean up and reference secrets stored in cbi pass store

### PR DESCRIPTION
**Pull request by netomi**

### CODE REVIEW

**Comment by netomi** [comment]
The diff view does not show anything as the support for secrets is not yet complete. Otterdog can currently import them but does not consider them during the plan / apply operations.

File: otterdog/adoptium.jsonnet [hunk 1]
```diff
@@ -115,6 +129,14 @@ orgs.newOrg('adoptium') {
         "hacktoberfest"
       ],
       web_commit_signoff_required: false,
+      secrets: [
```
Comment (gdams) [inline otterdog/adoptium.jsonnet:132]: These are no longer needed (delete them both please)

File: otterdog/adoptium.jsonnet [hunk 2]
```diff
@@ -149,6 +171,26 @@ orgs.newOrg('adoptium') {
...
+        orgs.newRepoSecret('AZURE_CLIENT_ID') {
+          value: "********",
+        },
+        orgs.newRepoSecret('AZURE_CLIENT_ID_OIDC') {
+          value: "********",
+        },
+        orgs.newRepoSecret('AZURE_CLIENT_SECRET') {
+          value: "********",
+        },
+        orgs.newRepoSecret('AZURE_CREDENTIALS') {
```
Comment (gdams) [inline otterdog/adoptium.jsonnet:184]: please remove this one

File: otterdog/adoptium.jsonnet [hunk 3]
```diff
@@ -583,6 +646,14 @@ orgs.newOrg('adoptium') {
         "hacktoberfest"
       ],
       web_commit_signoff_required: false,
+      secrets: [
+        orgs.newRepoSecret('PASSWORD') {
+          value: "********",
+        },
+        orgs.newRepoSecret('USERNAME') {
```
Comment (gdams) [inline otterdog/adoptium.jsonnet:653]:
```suggestion
        orgs.newRepoSecret('ARTIFACTORY_USER') {
```

File: otterdog/adoptium.jsonnet [hunk 4]
```diff
@@ -583,6 +646,14 @@ orgs.newOrg('adoptium') {
         "hacktoberfest"
       ],
       web_commit_signoff_required: false,
+      secrets: [
+        orgs.newRepoSecret('PASSWORD') {
```
Comment (gdams) [inline otterdog/adoptium.jsonnet:650]:
```suggestion
        orgs.newRepoSecret('ARTIFACTORY_PASSWORD') {
```

File: otterdog/adoptium.jsonnet [hunk 5]
```diff
@@ -214,6 +256,11 @@ orgs.newOrg('adoptium') {
           secret: "pass:bots/adoptium/github.com/ci-webhook-secret",
         },
       ],
+      secrets: [
+        orgs.newRepoSecret('JENKINS_WORKFLOW_SECRET') {
```
Comment (gdams) [inline otterdog/adoptium.jsonnet:260]: not sure what the value of this is, ping @smlambert can someone provide @netomi with the actual value?

**Comment by netomi** [comment]
I will push the initial import to main, rebase this PR and make the suggested changes, that way we see in the diff view the exact things that the PR will change.

**Review by gdams — APPROVED** [review]

**Review by fperezel — COMMENTED** [review]
There are some secrets that are used by a repo. So, in my opinion, they should remove.

File: otterdog/adoptium.jsonnet [hunk 6]
```diff
@@ -27,16 +27,16 @@ orgs.newOrg('adoptium') {
...
     },
     orgs.newOrgSecret('ADOPTIUM_BOT_TOKEN') {
-      value: "********",
+      value: "pass:bots/adoptium/github.com/project-token",
     },
     orgs.newOrgSecret('ADOPTIUM_TEMURIN_BOT_TOKEN') {
-      value: "********",
+      value: "pass:bots/adoptium.temurin/github.com/project-token",
     },
     orgs.newOrgSecret('SLACK_WEBHOOK_CODEFREEZE_URL') {
```
[the remark this answers is not in the document]
Comment (netomi) [reply otterdog/adoptium.jsonnet:38]: it is: https://github.com/adoptium/.github/blob/3e7e1d6586ae39c55702ede4a0e7fd87561ddffa/.github/workflows/code-freeze.yml#L7

File: otterdog/adoptium.jsonnet [hunk 7]
```diff
@@ -27,16 +27,16 @@ orgs.newOrg('adoptium') {
   ],
   secrets+: [
     orgs.newOrgSecret('ADOPTIUM_AQAVIT_BOT_TOKEN') {
```
[the remark this answers is not in the document]
Comment (netomi) [reply otterdog/adoptium.jsonnet:29]: this PR adds support for secrets as managed by otterdog. For this purpose we import the current status quo and adjust as needed. If the project concludes that a certain secret is not needed anymore, it should be removed, as we did for some secrets already with this PR.

[the remark this answers is not in the document]
Comment (gdams) [reply otterdog/adoptium.jsonnet:29]: We've always kept this there in case it's needed. Let's leave it there for now

**Review by gdams — APPROVED** [review]

**Comment by netomi** [comment]
@gdams before I can apply the changes, the jmc-build repo will need to use the updated secret names:

https://github.com/adoptium/jmc-build/blob/master/.github/workflows/build.yml#L97

**Comment by gdams** [comment]
> @gdams before I can apply the changes, the jmc-build repo will need to use the updated secret names:
> 
> 
> 
> https://github.com/adoptium/jmc-build/blob/master/.github/workflows/build.yml#L97

Feel free to apply them now, I'll make the changes

**Comment by netomi** [comment]
changes have been applied and I verified manually that the newly named secrets have been created.

**Pull request merged**